src/sagemaker/jumpstart/utils.py

```diff
@@ -60,6 +61,12 @@ def get_jumpstart_content_bucket(region: str) -> str:
     Raises:
         RuntimeError: If JumpStart is not launched in ``region``.
     """
+
+    if (
+        constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE in os.environ
+        and len(os.environ[constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE]) > 0
+    ):
+        return os.environ[constants.ENV_VARIABLE_JUMPSTART_CONTENT_BUCKET_OVERRIDE]
```

Comment:
question: do you think we should log a warning if there is an override?

Reply:
It's not really a warning -- i.e. we shouldn't attach negative connotation to this. Maybe an `INFO`?

Reply:
sounds good, i agree an `INFO` log is more appropriate